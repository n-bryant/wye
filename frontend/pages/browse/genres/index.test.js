import { GenresPage } from "./index";

describe("GenresPage", () => {
  const props = {
    calsses: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    const wrapper = shallow(<GenresPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
