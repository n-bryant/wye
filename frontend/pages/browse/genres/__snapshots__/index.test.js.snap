// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresPage should render successfully 1`] = `
<div
  className="wye-GenresPage"
>
  <BackgroundQuery
    useDefault={true}
  >
    <GenresQuery />
  </BackgroundQuery>
</div>
`;
